nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/ToggleBits.java

```diff
@@ -0,0 +1,35 @@
+package org.nd4j.linalg.api.ops.custom;
+
+import lombok.NonNull;
+import org.nd4j.autodiff.samediff.SDVariable;
+import org.nd4j.autodiff.samediff.SameDiff;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.api.ops.DynamicCustomOp;
+
+public class ToggleBits extends DynamicCustomOp {
+
+    public ToggleBits() {super();}
+
+    public ToggleBits(@NonNull INDArray input, @NonNull INDArray output) {
+        this(input);
+        addOutputArgument(input);
```

Comment:
Output may be null

Reply:
Null check for output - and incorrectly passing input array here, not output array